internal/accounts/accounts.go

```diff
@@ -18,9 +19,13 @@ func InitializeAccountStore(ctx context.Context, db sql.DB) error {
 	}
 	defer sp.Rollback(ctx)
 
-	err = initTables(ctx, sp)
+	upgradeFns := map[int64]versioning.UpgradeFunc{
+		-1: initTables,
```

Comment:
So this is a weird quirk (and is why I previously had the `-1` field be optional in the upgrader).

What we have defined here is our "first version" for our account store; essentially, our v0. No matter what changes we have in the future, this will always be our v0. Therefore, I think it just makes more sense for the original initialization to be a v0.